sdk/containers/azcontainerregistry/autorest.md

```diff
@@ -457,3 +457,25 @@ directive:
     transform: >
         $.ArtifactOperatingSystem.description = "The artifact platform's operating system.";
 ```
+
+### Remove useless Marshal method
```

Comment:
Why aren't these needed?

Reply:
These models will only be used in responses. Two ways to increase test coverage for the serialization part:
1. add useless marshal test
2. remove marshal function
And I choose second.